wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -467,11 +467,11 @@ private void checkWorkspaceIsRunningOrStarting(WorkspaceImpl workspace) throws C
     }
   }
 
-  private void removeWorkspaceQuietly(Workspace workspace) {
+  private void removeWorkspaceQuietly(String workspaceId) {
     try {
-      workspaceDao.remove(workspace.getId());
+      workspaceDao.remove(workspaceId);
     } catch (ServerException x) {
-      LOG.error("Unable to remove temporary workspace '{}'", workspace.getId());
+      LOG.error("Unable to remove temporary workspace '{}'", workspaceId);
```

Comment:
you're not the author of this code but still, it may be useful to add exception or message to logger.